master/reference/node/configuration.md

```diff
@@ -0,0 +1,50 @@
+---
+title: Configuring calico/node
+---
+
+The `calico/node` container is primarily configured through environment variables.
+
+
+## Environment Variables
+
+| Environment | Description                              | Schema | Default |
+| ------------- | -------- | ------ | ----- |
+| NODENAME | Node ID. | string | |
+| NO_DEFAULT_POOLS | Prevents calico from creating a default pool if one does not exist. | boolean | false |
+| HOSTNAME | Hostname. | string | |
+| IP | IP address which calico will bind to. | ip address | 10.0.0.1 |
+| IP6 | IPv6 address which calico will bind to. | ipv6 address | fed80::1 |
+| AS | AS number for this node. | int | global AS number |
+| DATASTORE_TYPE | Type of datastore. |can be `kubernetes` or `etcd` | etcd |
+| WAIT_FOR_DATASTORE | Prevent calico/node startup from initiating etcd connection until an etcd connection is made. | boolean | false |
+| CALICO_LIBNETWORK_ENABLED | Enables running the docker-libnetwork plugin directly in the calico/node container. | boolean | true |
```

Comment:
We should also discuss the volume mounts that are required for this container.   Although I'd be happy with this being a separate PR :-)